README.md

```diff
@@ -1,35 +1,78 @@
-# Gluco Check
+# Welcome to Gluco Check
 
-formerly Nightscout Assistant / Nightscout Status
+#### formerly [Nightscout Status](https://github.com/nielsmaerten/nightscout-assistant) 
+[![CodeFactor Grade](https://img.shields.io/codefactor/grade/github/nielsmaerten/gluco-check/main?style=flat-square)](https://www.codefactor.io/repository/github/nielsmaerten/gluco-check/)
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/nielsmaerten/gluco-check/Build,%20Test,%20Lint,%20Deploy?style=flat-square)](https://github.com/nielsmaerten/gluco-check/actions?query=workflow%3A%22Build%2C+Test%2C+Lint%2C+Deploy%22)
+[![Codecov](https://img.shields.io/codecov/c/github/nielsmaerten/gluco-check?style=flat-square)](https://codecov.io/gh/nielsmaerten/gluco-check)
 
-## Hey Google, talk to Gluco Check
+## Getting started
 
-> "Blood sugar is 102 and rising slowly as of 4 minutes ago."
+<!-- TODO: Update URL to Production version -->
+1. [Link your Nightscout site](https://nightly.diabase.app)
+2. [Ask for beta access](https://gitter.im/nielsmaerten)
+3. Say: **"Hey Google, talk to Gluco Check"**  
 
-## Bèta testing is now open
+## Things you can ask Gluco Check
+
+### Blood sugar
+
+Hey Google, ask Gluco Check my blood sugar
+
+### Insulin on Board / Carbs on Board
+
+Hey Google, ask Gluco Check how many carbs I have  
+Hey Google, ask Gluco Check my IOB  
+
+### Sensor age / Infusion set age
+
+Hey Google, ask Gluco Check when I last changed my sensor  
+Hey Google, ask Gluco Check my cannula's age
```

Comment:
Hey Google, ask Gluco Check how old my pump is
Hey Google, ask Gluco Check how old my pod is

Unsure about these as they are quite pod specific, but can say that at least in our lives, we often think of the pod and the pump and the cannula as the same thing (despite the fact that they very very clearly are not ha ha)

Reply:
That's interesting! I hadn't thought of this at all. (Probably because I have a 'belt' pump with a long tube)  

Currently, _infusion set_ & _cannula_ are synonyms. So when you say: "how old is my X", Google can substitute X for things like: sensor, dexcom, pod, libre, infusion set, cannula, ...

The only thing I'm a worried about is adding 'pump' as a synonym... because that might cause confusion with 'pump battery' and 'pump reservoir level'. That's because _X_ is a 'Metric'. So it can be replaced with "blood sugar", "carbs" and so on.

"Hey Google, how old is my carbs" is a query that doesn't make sense... but it would work. Because to the Assistant's Language Model, 'cannula age' and 'pump battery' are equal concepts ie Metrics.

Reply:
Ah okay, totally agree now that you explain the structure, seems better to leave "pump" out of it. Maybe just "pod" as an additional synonym then?

PS if you want to get _very_ specific, the kid-friendly terms we picked up from our clinic team were "toby" for pods (because the mascot they use for in their marketing here is "toby the turtle"), and "dexy" for the dexcom, and my understanding is that these are pretty common at least among the other little kids in the practice we go to. Heck, we have a clinic appointment coming up next week, if you are interested I could ask them if there are any other super-common terms they are aware of.